### PR TITLE
Replicate the creation of the gw-ci cycling test

### PR DESCRIPTION
**Pull request by guillaumevernieres**
It's useful to me so I suspect it could be useful to others. This PR adds 3 ctests that run the `g-w` `create_experiment.py` script for :
- Aero/Snow JEDI DA
- Atmos JEDI DA
- WCDA (SOCA + GSI)

each ctest will create an exp directory with a `COMROOT` and `EXPDIR`, the user can then cycle the experiment with `rocoto`.

I stopped at the creation of the experiment but this could be pushed further if we wanted too ...

### CODE REVIEW

**Review by CoryMartin-NOAA — DISMISSED** [review]
Looks good, thanks!

File: test/gw-ci/create_exp.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+expyaml="$1"
+export pslot="$2"
+HOMEgfs="$3"
+export RUNTESTS="$4"/${pslot}
+export ICSDIR_ROOT=/scratch1/NCEPDEV/global/glopara/data/ICSDIR
```
Comment (CoryMartin-NOAA) [inline test/gw-ci/create_exp.sh]: This is Hera specific, can we support other platforms too?

File: test/gw-ci/create_exp.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+export SLURM_ACCOUNT="da-cpu"
```
Comment (CoryMartin-NOAA) [inline test/gw-ci/create_exp.sh:6]: What is this used for? This test just runs a script, it doesn't need slurm does it?

File: test/gw-ci/create_exp.sh (hunk 1 above)
Comment (guillaumevernieres) [reply test/gw-ci/create_exp.sh]: arf ... will add the other machines

File: test/gw-ci/create_exp.sh (hunk 2 above)
Comment (guillaumevernieres) [reply test/gw-ci/create_exp.sh:6]: no, but the intent is for the user to run what was created, what comes out of the `g-w` config is not what we want.

**Review by CoryMartin-NOAA — COMMENTED** [review]
Instead of removing it, can we have the cmake define ICSDIR based on the machine it was built on?

**Comment by guillaumevernieres** [comment]
> Instead of removing it, can we have the cmake define ICSDIR based on the machine it was built on?

this is done under the hood in the g-w

**Review by CoryMartin-NOAA — DISMISSED** [review]

**Review by danholdaway — DISMISSED** [review]

**Review by CoryMartin-NOAA — DISMISSED** [review]

**Comment by guillaumevernieres** [comment]
> > Instead of removing it, can we have the cmake define ICSDIR based on the machine it was built on?
> 
> this is done under the hood in the g-w

I thought I made sure I didn't have icsdir_root defined in the shell ... But apparently not, ICSDIR_ROOT needs to be defined.

**Review by AndrewEichmann-NOAA — APPROVED** [review]
looks reasonable

**Pull request merged**